### PR TITLE
Add&remove

### PR DESCRIPTION
**Pull request by ZahraArshia**
Hello 😊 
## in this step of project:
- a new JavaScript file is added with the name of CRUD.js including class "Task" with methods for add, remove, edit and update
- index key of each task is updating according to its position in storage array
- edit function don't affect the position of the item in list, just changes the description
## in the feature:
the `clear all completed` button is disable for now, in next step its functionality will be implemented according to Microverse schedule

### CODE REVIEW

File: src/CRUD.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  editMood = false;
+
+  add() {
+    const temp = new Task(this.taskList.length, document.getElementById('inputDescription').value, false);
+    this.taskList.push(temp);
+    localStorage.setItem('taskList', JSON.stringify(this.taskList));
+    this.taskList = JSON.parse(localStorage.getItem('taskList'));
+    this.update();
+    location.reload();
+  }
```
Comment (micheaol) [inline src/CRUD.js]: Please ensure that `items` are numbered from ` 1 ` according to the project requirement and NOT ` 0 `.
image.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @ZahraArshia , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ZahraArshia** [comment]
Hello;
the indexes are starting from 1 now 😊

**Review by rustam-tolipov — APPROVED** [review]
## Hi @ZahraArshia   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**